### PR TITLE
Edit version json instead of maintaining our own

### PR DESCRIPTION
**Pull request by altrisi**
Launches.

Fixes IrisShaders/Iris-Installer-Maven#1 for installer versions with this.

### CODE REVIEW

**Comment by IMS212** [comment]
This seems to break the Install as Fabric Mod option, based off a quick look.

**Comment by altrisi** [comment]
Oh lol I'm dumb

**Comment by altrisi** [comment]
There it goes

File: src/main/java/net/hypercubemc/iris_installer/VanillaLauncherIntegration.java [hunk 1]
```diff
@@ -43,7 +45,27 @@ public static void installVersion(Path mcDir, String gameVersion, String loaderN
...
+            editVersionJson(profileJson);
+        }
+        Utils.writeToFile(profileJsonPath, profileJson.toString());
+    }
+    
+    private static void editVersionJson(Json profileJson) {
+        Json.Factory factory = Json.factory();
+        Map<String, Json> json = profileJson.asJsonMap();
+        // Replace fabric-loader-etc with iris-fabric-loader-etc
+        json.compute("id", (ignored, existing) -> factory.string("iris-"+existing.asString()));
```
Comment (Justsnoopy30) [inline src/main/java/net/hypercubemc/iris_installer/VanillaLauncherIntegration.java]: Make sure to add spacing between string concatenation

**Review by Justsnoopy30 — APPROVED** [review]
Looks good to merge

**Pull request merged**